build.sbt

```diff
@@ -22,7 +25,23 @@ val javacSettings = Seq(
   "-Xlint:unchecked"
 )
 
-lazy val commonSettings = mimaDefaultSettings ++ Seq(
+lazy val mimaSettings = mimaDefaultSettings ++ Seq(
+  mimaBinaryIssueFilters ++= Seq(
+    // Added to have better parity between Java and Scala APIs
+    ProblemFilters.exclude[ReversedMissingMethodProblem]("play.libs.ws.StandaloneWSRequest.getBody"),
```

Comment:
I guess the justification for the breaking change is that we don't support custom implementations of the WS API? i.e. you have to use our provided AhcWSRequest and can't write your own.